src/visualization/Turi Create Visualization/src/user_interface/package.json

```diff
@@ -12,8 +12,8 @@
         "react": "^16.2.0",
         "react-dom": "^16.2.0",
         "react-json-pretty": "^1.7.5",
-        "vega": "^3.2.1",
-        "vega-embed": "^3.2.0",
+        "vega": "4.2.0",
+        "vega-embed": "3.18.2",
         "vega-tooltip": "^0.6.1"
```

Comment:
Do we still need vega-tooltip?

Reply:
Yeah for now with the implementation we have, no issues there, but if we need to change it, it should be a separate PR

Reply:
So we do still need it, but our existing version is still compatible?

Reply:
yeah, we still need it and our existing version is compatible.